tools/orxFontGen/build/premake4.lua

```diff
@@ -104,27 +104,18 @@ solution "orxFontGen"
 
     includedirs
     {
-        "../include",
-        "../../../code/include",
         "$(ORX)/include",
-        "../../../extern/stb_image",
-        "../../../extern/freetype/include"
+        "$(ORX)/../extern/stb_image",
+        "$(ORX)/../extern/freetype/include"
     }
 
     configuration {"not macosx"}
-        libdirs
-        {
-            "../lib",
-            "../../../code/lib/static",
-            "$(ORX)/lib/static"
-        }

```

Comment:
Only the macosx version is using dynamic libraries here. This is due to a constraint coming from XCode.

Reply:
Sorry, could you clarify here? I'll restore this section.